web/src/components/gnd-feature-type-editor/gnd-form-element-editor.js

```diff
@@ -143,9 +146,9 @@ class GndFormElementEditor extends React.Component {
             onChange={ev => this.handleTypeChange(ev.target.value)}
             onBlur={ev => this.handleTypeChange(ev.target.value.trim())}
           >
-            <MenuItem value="text_field">Text</MenuItem>
-            <MenuItem value="select_one">Select one</MenuItem>
-            <MenuItem value="select_multiple">Select multiple</MenuItem>
+            <MenuItem value="text_field"><ShortText style={{ marginRight: 5 }}></ShortText>Text</MenuItem>
```

Comment:
Can these inline styles be replaced with a CSS class instead?

Reply:
Done